README.md

```diff
@@ -335,6 +335,52 @@ export const getLoadContext: GetLoadContext = ({ context }) => {
 }
 ```
 
+## AsyncLocalStorage
+
+You can use AsyncLocalStorage, which is supported by Node.js, Cloudflare Workers, etc.
+You can easily store context using Hono's Context Storage Middleware.
+
+```ts
+// server/index.ts
+import { Hono } from 'hono'
+import { contextStorage } from 'hono/context-storage'
+
+export interface Env {
+  Variables: {
+    headers: Headers
+    key: string
+    // db: DatabaseConnection // It's also a good idea to store database connections, etc.
+  }
+}
+
+const app = new Hono<Env>()
+
+app.use(contextStorage())
+
+app.use(async (c, next) => {
+  c.set('headers', c.req.raw.headers)
+  c.set('message', 'Hello!')
+
+  await next()
+})
+
+export default app
+```
+
+You can retrieve and process the context saved in Hono from Remix as follows:
+
+```ts
+// app/routes/_index.tsx
+import type { Env } from 'server'
+import { getContext } from 'hono/context-storage' // It can be called anywhere for server-side processing.
+
+export const loader = () => {
+  const cookie = getContext<Env>().var.headers.get('Cookie')
```

Comment:
I think the `message = ...` is very good, but the `cookie = ...`  is not necessary because you can get the `Headers` object by the default Remix feature. So how about removing this line, [the line](https://github.com/yusukebe/hono-remix-adapter/pull/47/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R350), and [the line](https://github.com/yusukebe/hono-remix-adapter/pull/47/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R361)?

Reply:
I tried to fix it like that.